src/golang/lib/collections/artifact/types.go

```diff
@@ -25,129 +19,21 @@ import (
 type Type string
 
 const (
-	TableType Type = "table"
-	FloatType Type = "float"
-	BoolType  Type = "boolean"
-	JsonType  Type = "json"
+	Untyped       Type = "untyped"
+	StringType    Type = "string"
+	BoolType      Type = "bool"
+	NumericType   Type = "numeric"
+	DictType      Type = "dictionary"
+	TupleType     Type = "tuple"
+	TabularType   Type = "tabular"
+	JsonType      Type = "json"
+	BytesType     Type = "bytes"
+	ImageType     Type = "image"
+	PicklableType Type = "picklable"
 )
 
-type specUnion struct {
-	Type  Type         `json:"type"`
-	Table *table.Table `json:"table,omitempty"`
-
-	// TODO(ENG-1119): The float artifact currently also represents integers.
```

Comment:
Should we cancel ENG-1119 then?

Reply:
Yeah I can this task should be deprecated.